### DISCUSSION TITLE
Published Packages does not show availability on NPM?

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#208] [General]
![grafik](https://github.com/user-attachments/assets/913acee4-0ce5-48ff-94ab-2a3369c9ccb6)

Either the [npm package](https://www.npmjs.com/package/@lichtblick/suite) is referenced there or in the readme?

**Comment by carolinabamorim** [comment]
Thank you @jdsika.
These packages are not related to NPM packages. If you check now you can see we already have our docker image on this package section.
 NPM packages are referred in the json of the project.
 I'll close this discussion.
 Thank you.
